homeassistant/components/onkyo/media_player.py

```diff
@@ -176,14 +177,14 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     host = config.get(CONF_HOST)
     hosts = []
 
-    def service_handle(service):
+    def service_handle(service: ServiceCall) -> None:
```

Comment:
`ATTR_ENTITY_ID` and `ATTR_HDMI_OUTPUT` are required by the schema:
https://github.com/home-assistant/core/blob/a8fe9f2eff70d83fd347e3d2fa94d9730ab77eaf/homeassistant/components/onkyo/media_player.py#L112-L117